lib/build-group-id.js

```diff
@@ -33,9 +33,12 @@ module.exports = function buildGroupId({
     }
   }
 
-  const cloakedMsgId = new CloakedMsgId(msgId, readKey).toString()
+  const cloakedMsgId = new CloakedMsgId(msgId, readKey)
+    .toString()
+    .replaceAll('/', '_')
+    .replaceAll('+', '-')
```

Comment:
Can we use https://github.com/joaquimserafim/base64-url `base64url.escape()`? I'm afraid that the more we sprinkle these replaceAll (across various repos), the more likely is that one day one of them will get a tiny detail wrong.

Reply:
switched to the library now. but had to add a = at the end because the library removed the padding

Reply:
actually there's this in uri2 but it's not exported https://github.com/ssbc/ssb-uri2/blob/ab051cbe99a7b87027305464d8ac6cb65c24943b/src/index.ts#L33-L41